programs/bpf_loader/src/syscalls.rs

```diff
@@ -90,6 +90,8 @@ pub enum SyscallError {
     CopyOverlapping,
     #[error("Return data too large ({0} > {1})")]
     ReturnDataTooLarge(u64, u64),
+    #[error("Hashing too many sequences")]
```

Comment:
Do we really need this new error?

Reply:
I don't know. `BpfError` is either `SyscallError` or `VerifierError`, so it seems for consistency it should be some of `SyscallError`s, none of existing ones seems to be adequate. There are other `SyscallError`s that are used only once.  Where's `BadParam` defined? I found only `CliError::BadParameter`

Reply:
Ok, sounds good